src/Content/Conversation.php

```diff
@@ -205,87 +252,48 @@ public function builtinActivityPuller(array $activity, array &$conv_responses)
 	public function formatActivity(array $links, string $verb, int $id): string
 	{
 		$this->profiler->startRecording('rendering');
-		$o        = '';
 		$expanded = '';
-		$phrase   = '';
 
-		$total = count($links);
-		if ($total == 1) {
-			$likers = $links[0];
+		$phrase = $this->getLikerPhrase($verb, $links);
+		$total  = count($links);
 
-			// Phrase if there is only one liker. In other cases it will be uses for the expanded
-			// list which show all likers
-			switch ($verb) {
-				case 'like':
-					$phrase = $this->l10n->t('%s likes this.', $likers);
-					break;
-				case 'dislike':
-					$phrase = $this->l10n->t('%s doesn\'t like this.', $likers);
-					break;
-				case 'attendyes':
-					$phrase = $this->l10n->t('%s attends.', $likers);
-					break;
-				case 'attendno':
-					$phrase = $this->l10n->t('%s doesn\'t attend.', $likers);
-					break;
-				case 'attendmaybe':
-					$phrase = $this->l10n->t('%s attends maybe.', $likers);
-					break;
-				case 'announce':
-					$phrase = $this->l10n->t('%s reshared this.', $likers);
-					break;
-			}
-		} elseif ($total > 1) {
-			if ($total < $this->config->get('system', 'max_likers')) {
-				$likers = implode(', ', array_slice($links, 0, -1));
-				$likers .= ' ' . $this->l10n->t('and') . ' ' . $links[count($links) - 1];
-			} else {
-				$likers = implode(', ', array_slice($links, 0, $this->config->get('system', 'max_likers') - 1));
-				$likers .= ' ' . $this->l10n->t('and %d other people', $total - $this->config->get('system', 'max_likers'));
-			}
+		if ($total > 1) {
+			$spanatts  = "class=\"btn btn-link fakelink\" onclick=\"openClose('{$verb}list-$id');\"";
+			$explikers = $phrase;
 
-			$spanatts = "class=\"btn btn-link fakelink\" onclick=\"openClose('{$verb}list-$id');\"";
-
-			$explikers = '';
 			switch ($verb) {
 				case 'like':
-					$phrase    = $this->l10n->t('<button type="button" %1$s>%2$d people</button> like this', $spanatts, $total);
-					$explikers = $this->l10n->t('%s like this.', $likers);
+					$phrase = $this->l10n->t('<button type="button" %1$s>%2$d people</button> like this', $spanatts, $total);
```

Comment:
This also should be a `tt()` call, even if we never use the singular version. The translation string doesn't know the programming context introduced by the condition on `$total`